com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/HDRenderPipeline.PostProcess.cs

```diff
@@ -1713,7 +1713,7 @@ class TemporalAntiAliasingData
             Vector4 scales = new Vector4(historyRenderingViewport.x / prevHistory.rt.width, historyRenderingViewport.y / prevHistory.rt.height, mainRTScales.x, mainRTScales.y);
             passData.taaScales = scales;
 
-            passData.finalViewport = (TAAU || runsAfterUpscale) ? camera.finalViewport : new Rect(0, 0, RTHandles.rtHandleProperties.currentViewportSize.x, RTHandles.rtHandleProperties.currentViewportSize.y);
+            passData.finalViewport = (upscalingEnabled || runsAfterUpscale) ? camera.finalViewport : new Rect(0, 0, RTHandles.rtHandleProperties.currentViewportSize.x, RTHandles.rtHandleProperties.currentViewportSize.y);
```

Comment:
These `upscalingEnabled || runsAfterUpscale` checks are redundant for DLSS, here we want to check TAAU explicitly

Reply:
The upscalingEnabled variable seems redundant, since we already have the runsAfterUpscale one. So in all cases above I would replace upscalingEnabled with runsAfterUpscale. Or perhaps I have missed something?